_shows/17_18/replay.md

```diff
@@ -49,6 +49,15 @@ assets:
 
 prod_shots: vzmkMh
 
+links:
+  - type: Review
+    href: http://kevcastletheatrereviews.blogspot.co.uk/2017/12/replay-by-nicola-wren-nottingham-new.html
+    snapshot: 6nf8f
+    publisher: Kev Castle Reviews
+    title: "\"Replay\" by Nicola Wren"
+    date: 2018-03-16
```

Comment:
Definitely not a 2018 show ;)

Reply:
Look, we all know dating's not my strong point alright